_posts/2017-03-23-first-ocim-blog.markdown

```diff
@@ -0,0 +1,107 @@
+
+Next up was Roi Lipman from Redis Labs, who explained how he had developed the Redis graph module based on a hexastore model, where node-relationship-node triplets were stored in six permutations to enable fast prefix-based searches.
+
+One idea that had come up during prior meetings with several interested parties was that of a shared standard of internal graph query representation, possibly compiled from several distinct source languages.
+Stefan Plantikow gave an insight to how Neo and the Cypher Language Group had been thinking about such a model, called QUIL (Query Intermediary Language).
+
+Tomasz Zdybał from Czech Dgraph Labs presented Dgraph, an in-memory native graph database, and their implementation of a graph query language based on Facebook's GraphQL in their product.
+Tomasz highlighted Dgraph's intentions of adding support for Cypher, and how schema validation was an important topic.
+
+Just before lunch, Alastair Green took the floor again, standing in for BitNine, the Korean company behind Agens Graph, discussing how Cypher, SQL, and other query languages may be integrated with one another.
+BitNine were originally scheduled for attendance at the oCIM, but had to decline, as the Central European location was unfit given time their time constraints.
+Alastair went through slides by Kisung Kim from BitNine, detailing the hybrid relational/graph architecture of Agens Graph, which is based on PostgreSQL.
+The most highlighted contribution was the way in which BitNine had introduced integration points between SQL and Cypher, allowing each as a subquery construct to the other, and how functions may be shared as expressions between the languages.
+
+---
+
+The lunch break paved way for the big topic previously brought up by Marcus Paradies of SAP in the morning: multiple graphs.
+Cypher has always been a language that operates on a single implicitly understood graph, and producing a stream or records as output (the collection of which effectively form a table).
+Alastair Green discussed motivations for changing this model to make Cypher a language _closed over graphs_, sketching out a future where Cypher would come to operate on a multitude of graphs, and also producing a multitude of graphs as output.
+Alastair touched on important subtopics such as identity, addressing, and how compositions of graphs from distinct sources may be defined.
+
+Stefan Plantikow led a longer session on the very topic, presenting his latest thinking on how Cypher can be remodeled towards a graph-in-graph-out paradigm.
+Stefan deep-dove into the motivations, but the focus was put on how the concept of multiple graphs could be made logically consistent, and how the execution model could be extended, while still paying respects to Cypher's sizable user base and imposing breaking change in semantics.
```

Comment:
"Stefan deep-dove into the motivations, but the focus was put on how the concept of multiple graphs could be made logically consistent, and how the execution model could be extended"
-> 
"Stefan discussed the motivations from a different angle to those covered by Alastair, focusing on how to make the concept of multiple graphs logically consistent and how to extend the execution model"